src/Microsoft.Extensions.Logging.Testing/LoggedTest/LoggedTest.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit.Abstractions;
+
+namespace Microsoft.Extensions.Logging.Testing
+{
+    public class LoggedTest : SinklessLoggedTest
+    {
+        // Obsolete but keeping for back compat
+        public LoggedTest(ITestOutputHelper output = null) : base (output) { }
+
+        public ITestSink TestSink { get; set; }
+
+        public override void AdditionalSetup()
+        {
+            base.AdditionalSetup();
```

Comment:
This means we will need to update the implementation in Kestrel at https://github.com/aspnet/KestrelHttpServer/blob/2d51d236979ee731b8675e5879a56b708ec2fa4b/test/Kestrel.FunctionalTests/TestHelpers/TestApplicationErrorLoggerLoggedTest.cs#L12